<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallsFinder</class>
 <widget class="QWidget" name="CallsFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Old Calls</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>781</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="fromDateEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumDate">
    <date>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="fromLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>From:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="toDateEdit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumDate">
    <date>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="toLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>To:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Look For Old Registered Calls</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>fromDateEdit</tabstop>
  <tabstop>toDateEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
